### DISCUSSION TITLE
Valvole termostatiche eqiva eQ-3

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#2096] [HUB: Home Assistant]
Interessato al risparmio, ho provato l'integrazione di questa valvola termostatica bluetooh secondo la guida
https://indomus.it/guide/integrare-una-testa-termostatica-eqiva-eq-3-bluetooth-a-home-assistant/

Purtroppo la valvola non viene vista dal raspberry, con l'app android e telefonino vedo il bluetooh e il MACaddress, la valvola è vicina al raspy ma risulta 'unavailable' (come allegato)

C'è qualche novità da impostare, dato che la guida credo sia di un po' di tempo fa? grazie 

![Schermata del 2022-11-22 18-45-34](https://user-images.githubusercontent.com/84615011/203384959-da7c9f98-b25d-4029-83ef-9a3d84181ad9.png)

**Comment by 50494554524F** [comment]
se la testa è vicina al raspberry prova ad andare indietro con le versioni di homeassistant, stanno rimettendo mano al bluetooth ed è probabile che ci sia qualcosa che non funzioni....

**Reply by calz64** [reply]
Andare indietro significa fare un downgrede del core? ci vorrebbe un bkup di un po' di versioni fa? mi sa che non ho la possibilità.

**Reply by 50494554524F** [reply]
puoi provare in macchina virtuale... (te lo dico perché da quando è cominciato il "restyling" ho avuto grandi problemi ad usare switchbot ed ora sembrano corretti in 2022.11. penso potesti provare con OS 7.6 se lo usi e 2022.6, l'ultima versione che non mi ha dato problemi)
